godtools/App/Share/SwiftUI Views/GTModalView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  GTModalView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/3/25.
+//  Copyright © 2025 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct GTModalView<Content: View>: View {
```

Comment:
I like this.  It would be good to use this for the opt-in notification view and tool settings view.  I'm going to create a ticket for that.